### DISCUSSION TITLE
How to link listening progress of media to a different server URL

### DISCUSSION THREAD

**Discussion by DavidPesticcio** [advplyr/audiobookshelf#2656] [Q&A] [answered]
Hi all,

Since testing Audiobookshelf with the Android app using an initial server URL of http://192.168.x.x:13378, a production URL is now being used, but I get the below error message in the app.

This is the message I see when connected to the :
```
"MessageMediaLinkedToADifferentServer": "Media is linked to an Audiobookshelf server on a different address ({0}). Progress will be synced when connected to this server address.",
```
Ref for message: https://github.com/advplyr/audiobookshelf-app/blob/7e3b3f55aef08c2f0f3d842e4c1a0a5178660cab/strings/no.json#L259

Ref for usage: https://github.com/advplyr/audiobookshelf-app/blob/7e3b3f55aef08c2f0f3d842e4c1a0a5178660cab/pages/item/_id/index.vue#L40

How can the listening progress be synced to a different server URL - instead of the initial server URL?

Many Thanks.

**Comment by advplyr** [comment, marked as answer]
You have to re-download the book on the updated server address. Abs does not yet support associating downloaded media with multiple different server addresses. That is planned but for now you need to delete and re-download the media.
If you have unsaved progress that you were listening to offline or while not connected you will have to re-connect to the server address the media was downloaded with so it syncs with the server.

**Reply by DavidPesticcio** [reply]
That worked great! Thanks.

Perhaps this could be added to the FAQ... :smiley::+1: 

### The steps I followed to resolve this situation:
1. Reconnect the app to original server that was used to download the book
2. Allow the progress to sync to the server
3. Go to the hamburger menu and select "Local Media"
4. Remove the relevant book
5. Disconnect from the original server
6. Connect to the new server
7. Download the same book again to Local Media
8. Continue using the app with the new server as usual
